js/data/currencies.js

```diff
@@ -651,6 +651,10 @@ function getIndexedCurrencies() {
   return _indexedCurrencies;
 }
 
+export function getListofCurrencies() {
+  return _.pluck(currencies, 'code');
+}
```

Comment:
I'll take this out, looking at the code again I was already validating for an undefined currency, and shouldn't be checking getCurrencyByCode if the currencyCode isn't set.